core/src/config/api.ts

```diff
@@ -30,7 +33,29 @@ export const DEFAULT = {
       // enable action response to logger
       enableResponseLogging: false,
       // params you would like hidden from any logs
-      filteredParams: ["password", "passwordHash", "csrfToken", "private_key"],
+      // api.plugins.plugins.map(p => p.apps).filter(a => a).flat().map(a => a.options).flat().filter(o => o.required).map(o => o.key)
+      filteredParams: function () {
+        const filteredParams = [
+          "password",
+          "passwordHash",
+          "csrfToken",
+          "private_key",
+        ];
```

Comment:
we could add ery common ones here liek `apiKey`, `token`, `apiToken`, `secret`

Reply:
Made some changes so we can use `type=password` directly